core/templates/dev/head/components/state/response_header_directive.html

```diff
@@ -80,9 +80,12 @@
     <em ng-if="getOutcome().newStateName">(<[getOutcome().newStateName]>)</em>
     <em ng-if="!getOutcome().newStateName">Nowhere yet...</em>
   </span>
-  <span ng-if="getOutcome() && (isOutcomeLooping() || (!isCorrect() && !getOutcome().dest && isInQuestionMode()))">
+  <span ng-if="getOutcome() && isOutcomeLooping() && !isInQuestionMode()">
     <em>(try again)</em>
   </span>
+  <span ng-if="isInQuestionMode() && getOutcome() && !isCorrect() && !getOutcome().dest">
+    <em>Wrong<br> (try again)</em>
```

Comment:
I think we can drop the (Try again) part as from the creators' perspective, the answer is either correct or wrong, right? The learners' would need to try again.

Reply:
Yeah, but Wrong Answer could also mean the learner's answer is marked wrong (maybe in an overall evaluation) and they move on to next question, whereas here, it is considered wrong, and the learner stays in the same question. What do you think?

Reply:
Let's keep as-is for now. It may need to be changed later but I think that depends a bit on the subsequent work of the Practice Questions team.